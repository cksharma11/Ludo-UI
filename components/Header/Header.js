import React from 'react';
import PropTypes from 'prop-types';
import HeaderStyles from './Header.style';

const Header = ({ title }) => {
  return (
    <div className="header">
      {title}
      <style jsx>{HeaderStyles}</style>
    </div>
  );
};

Header.propTypes = {
  title: PropTypes.string.isRequired
};

export default Header;
