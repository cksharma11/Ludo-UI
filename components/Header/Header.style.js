import css from 'styled-jsx/css';
import colors from '../../config/styles/colors';

const HeaderStyles = css`
  .header {
    font-family: fantasy;
    font-size: 150px;
    text-align: center;
    width: 100%;
    height: 200px;
    background: ${colors.gray.shade1};
    color: ${colors.orange.shade1};
  }
`;

export default HeaderStyles;
