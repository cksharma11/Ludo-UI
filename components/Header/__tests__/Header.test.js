import React from 'react';
import { shallow } from 'enzyme';
import Header from '../Header';

describe('Header', () => {
  it('should render properly', () => {
    const wrapper = shallow(<Header title="title" />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
