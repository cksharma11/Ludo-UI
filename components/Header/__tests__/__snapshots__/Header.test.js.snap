// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render properly 1`] = `
"<div className=\\"jsx-1835151374 header\\">
  title
  <JSXStyle id=\\"1835151374\\">
    .
    h
    e
    a
    d
    e
    r
    .
    j
    s
    x
    -
    1
    8
    3
    5
    1
    5
    1
    3
    7
    4
    {
    f
    o
    n
    t
    -
    s
    i
    z
    e
    :
    3
    0
    p
    x
    ;
    t
    e
    x
    t
    -
    a
    l
    i
    g
    n
    :
    c
    e
    n
    t
    e
    r
    ;
    p
    a
    d
    d
    i
    n
    g
    :
    1
    0
    p
    x
    ;
    c
    o
    l
    o
    r
    :
    b
    l
    a
    c
    k
    ;
    h
    e
    i
    g
    h
    t
    :
    5
    v
    h
    ;
    b
    a
    c
    k
    g
    r
    o
    u
    n
    d
    :
    r
    a
    d
    i
    a
    l
    -
    g
    r
    a
    d
    i
    e
    n
    t
    (
     
    e
    l
    l
    i
    p
    s
    e
     
    a
    t
     
    c
    e
    n
    t
    e
    r
    ,
     
    r
    g
    b
    (
    9
    9
    ,
    1
    5
    5
    ,
    1
    5
    5
    )
     
    0
    %
    ,
     
    r
    g
    b
    (
    2
    5
    ,
    4
    1
    ,
    4
    1
    )
     
    9
    9
    %
    ,
     
    r
    g
    b
    (
    6
    9
    ,
    1
    0
    8
    ,
    1
    0
    8
    )
     
    1
    0
    0
    %
     
    )
    ;
    }
  </JSXStyle>
</div>"
`;
